src/Hit/Git.hs

```diff
@@ -42,21 +42,28 @@ runFresh (nameOrMaster -> branch) = do
     "git" ["rebase", "origin/" <> branch]
 
 -- | @hit new@ command.
-runNew :: Int -> IO ()
-runNew issueNum = do
+runNew :: Int -> Bool -> IO ()
+runNew issueNum assignOwner = do
     login <- getUsername
     let issueId = mkIssueId issueNum
-    issueTitle <- getIssueTitle issueId
-    let shortDesc = mkShortDesc issueTitle
+    issue <- fetchIssueStrict issueId
```

Comment:
Btw, if `getIssueTitle` function is not used anymore, I guess it's safe to remove it. We can implement it later when we need it again, it's just one line.